articles/install-guide/csinstall.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Develop with Q# + C#
+author: natke
+ms.author: nakersha
+ms.date: 9/30/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.cs
+---
+# Develop with Q# + C#
+
+Install the QDK to develop C# host programs to call Q# operations.
+
+Q# is a member of the .NET ecosystem and therefore is built to play well with other .NET languages--specifically C#. You can work with this pairing inside different development environments:
+
+- Q# + C# using Visual Studio (Windows)
+- Q# + C# using Visual Studio Code (Windows, Linux and Mac)
+- Q# + C# using the `dotnet` command-line tool
+
+## Develop with Q# + C# using Visual Studio
+
+Visual Studio offers a rich environment for developing Q# programs. The Q# Visual Studio extension contains templates for Q# files and projects as well as syntax highlighting, code completion and IntelliSense support.
+
+
+1. Pre-requisites
+
+    - [Visual Studio](https://visualstudio.microsoft.com/downloads/) 16.3, with the .NET Core cross-platform development workload enabled
+
+1. Install the Q# Visual Studio extension
+
+    - Download the [Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit)
+    - Add the extension to Visual Studio
```

Comment:
I would replace these two instructions with "Download and install the Visual Studio Code extension" (keeping the link)

Reply:
Done in https://github.com/geduardo/quantum-docs-pr/commit/1fa68d9af90806f0ab9f40de19a76881199308f0